### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by leonardhyu**

### CODE REVIEW

**Comment by leonardhyu** [comment]
@ArtemConstantinov could you solve the conflict, especially the file about port L23?

File: xoa_driver/internals/core/commands/enums.py [hunk 1]
```diff
@@ -1796,23 +1831,460 @@ class TSNStatisticsTypes(IntEnum):
...
+    LT_RX_ERROR_STAT_0 = 0x2A
+    LT_RX_ERROR_STAT_1 = 0x2B
+    LT_RX_ANALYZER_CONFIG = 0x38
+    LT_RX_ANALYZER_TRIG_MASK = 0x39
+    LT_RX_ANALYZER_STATUS = 0x3A
+    LT_RX_ANALYZER_RD_ADDR = 0x3B
+    LT_RX_ANALYZER_RD_PAGE = 0x3C
+    LT_RX_ANALYZER_RD_DATA = 0x3D
+
+
```
Comment (ArtemConstantinov) [inline xoa_driver/internals/core/commands/enums.py:2289]: Must not be here, is an enum which have no usage in driver's core module

File: xoa_driver/functions/anlt.py [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+    :param lane: lane index, starting from 0
+    :type lane: int
+    :param emphasis: coefficient index (pre, pre2, pre3, main, post)
+    :type emphasis: str
+    :return:
+    :rtype: None
+    """
+    conn, mid, pid = port._conn, port.kind.module_id, port.kind.port_id
+    await commands.PL1_LINKTRAIN_CMD(conn, mid, pid, lane).set(
+        cmd=LinkTrainCmd.CMD_INC, arg=emphasis.value
```
Comment (ArtemConstantinov) [inline xoa_driver/functions/anlt.py:221]: used wrong value for parameter cmd

**Review by ArtemConstantinov — APPROVED** [review]

**Comment by ArtemConstantinov** [comment]
Run more tests, and after the code can be merged.

**Pull request merged**